System/Atom.cpp

```diff
@@ -549,7 +550,16 @@ namespace RTE {
 				}
 				return hitStep;
 			}
-			RTEAssert(0, "Atom shouldn't be taking steps beyond the trajectory!" + (m_OwnerMO ? " Owner is " + m_OwnerMO->GetPresetName() + "." : ""));
+			std::string abortString = "Atom shouldn't be taking steps beyond the trajectory!";
```

Comment:
Should add a bunch of newlines here otherwise it's one long string of confusing garbage.